packages/nocodb/src/db/BaseModelSqlv2.ts

```diff
@@ -1621,7 +1621,16 @@ class BaseModelSqlv2 {
 
         tQb.select(jsonBuildObject);
 
-        selectors.push(this.dbDriver.raw(`(??) as ??`, [tQb, `${f.alias}`]));
+        if (this.dbDriver.client.config.client === 'mysql2') {
+          selectors.push(
+            this.dbDriver.raw(`JSON_UNQUOTE((??)) as ??`, [
```

Comment:
Does 2 brackets required here  ? `((??))`  => `(??)`

And string literal not required as well since no js variables used.